src/Commands/Down.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Stancl\Tenancy\Commands;
+
+use Illuminate\Console\Command;
+use Stancl\Tenancy\Concerns\HasATenantsOption;
+
+class Down extends Command
+{
+    use HasATenantsOption;
+
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+     protected $signature = 'tenancy:down
+         {--redirect= : The path that users should be redirected to}
+         {--retry= : The number of seconds after which the request may be retried}
+         {--refresh= : The number of seconds after which the browser may refresh}
+         {--secret= : The secret phrase that may be used to bypass maintenance mode}
+         {--status=503 : The status code that should be used when returning the maintenance mode response}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Put tenants into maintenance';
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        tenancy()->runForMultiple($this->option('tenants'), function ($tenant) {
```

Comment:
@lukinovec Does the `HasATenantsOption` trait *require* `--tenants` or make it optional?

If it's optional, then it's possible that `$this->option('tenants')` will return `null`. Which would run this for **all** tenants.

Which can be the desired behavior, and probably is the desired behavior, but it's not immediately clear from looking at the code. So if you can confirm what I wrote above, we probably want to add a comment like this:
```suggestion
    {
        // This runs for all tenants if no --tenants are specified
        tenancy()->runForMultiple($this->option('tenants'), function ($tenant) {
```

Reply:
And you can probably do the same in `Up.php`

Reply:
Yeah, the option is optional, and if not specified, the code runs for all tenants

Reply:
I added the comments